src/graph/graph.ts

```diff
@@ -71,23 +71,24 @@ export const updateGraphState = (player: Player, board: Board) => {
         const sibling = siblings[key];
 
         if (sibling !== null) {
-          graph
-            .addEdge(getTileName(tile), getTileName(sibling))
-            .addEdge(getTileName(sibling), getTileName(tile));
+          graph.addEdge(getTileName(tile), getTileName(sibling));
         }
       }
     });
   });
   return graph;
 };
 
-export const checkOppositePath = (graph): [] => {
+export const getOppositePath = (graph): string[] | [] => {
```

Comment:
You should probably type it like this.

```suggestion
export const getOppositePath = (graph: ReturnType<typeof Graph>): string[] | [] => {
```